docs/components/vxp-label.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebarDepth: 0
+---
+
+# VxpLabel
+## Live Demo
```

Comment:
should there be a little explanation what this component does?
There are explanations in other components here